src/style.css

```
@@ -0,0 +1,33 @@
+body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.stars--0,
+.stars--1,
+.stars--2,
+.stars--3,
+.stars--4,
+.stars--5 {
```

just add `stars` block and use it with corresponding modifier
```suggestion
.stars {
```